src/hooks/useCookieResyncOnResume.ts

```diff
@@ -0,0 +1,51 @@
+import { AppState, AppStateStatus } from 'react-native'
+import { useEffect } from 'react'
+
+import Minilog from '@cozy/minilog'
+import CozyClient, { useClient } from 'cozy-client'
+
+import { resyncCookies } from '/libs/httpserver/httpCookieManager'
+
+const log = Minilog('useGlobalAppState')
```

Comment:
Shouldn't it be 'useCookieResyncOnResume'?

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/d79a634f6a0e19820c433eb60a85f072b7327c74..d97e25177cceed3efb818a22c57d1d3212b358ba